### PR TITLE
opensearch-dashboards-2: New package

### PR DESCRIPTION
**Pull request by pdeslaur**
There are 3 known CVEs on the package - we'll follow up with advisories.

Version's policy: https://endoflife.date/opensearch

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: opensearch-dashboards-2.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      sed -i 's/\("osd:bootstrap": "scripts\/use_node scripts\/build_ts_refs\)\( && scripts\/use_node scripts\/register_git_hook\)/\1 --allow-root\2 --allow-root/' package.json
+
+  - runs: |
+      set -x
+
+      # Our commond LDFLAGS cause some issues when building for aarch64. We
+      # unset our global flags to allow the build to succeed.
+      unset LDFLAGS
+      yarn osd bootstrap --allow-root
+      yarn build-platform --linux-arm --skip-os-packages --release --allow-root
```
Comment (rawlingsj) [inline opensearch-dashboards-2.yaml:51]: I'm wondering if these flags need to change based on which arch is building. I.e x86 or arm. 

I can take a look shortly

**Review by rawlingsj — COMMENTED** [review]
Just the one comment which I can check if need

File: opensearch-dashboards-2.yaml (hunk 1 above)
Comment (rawlingsj) [inline opensearch-dashboards-2.yaml:51]: I also noticed that the plugins folder is empty..
```
ls build/opensearch-dashboards/plugins/
```

I'm guessing we'll need to build them too

Comment (rawlingsj) [reply opensearch-dashboards-2.yaml]: I'm not sure yet if we should expect plugins built into the ./plugins folder, let's come back to this.

**Review by dlorenc — APPROVED** [review]

**Pull request merged**